intro-section.tex

```diff
@@ -3,152 +3,272 @@ \section{Cylc Introduction}
 $ cylc validate hello_world
 $ cylc run hello_world
 \end{lstlisting}
 
-\paragraph*{Demo}
-To run the hello world example, create a directory called hello\_world. This
-will be our suite directory.
+\more{
+    \begin{itemize}
+        \item
+    \lstinline{cylc-register} just associates a name with a suite definition.
+    Commands that parse a suite can refer to this suite name or the file path;
+    commands that connect to a suite daemon must refer to the suite name.
+
+        \item
+    \lstinline{cylc validate} checks for errors in the suite.rc file.
+
+        \item
+    \lstinline{cylc run} starts a server program to manage your workflow. It
+    {\em daemonizes} (by default) and will stay alive if you log out. If your
+    suite runs to completion, it will automatically shut down.
+    \end{itemize}
+}
+
+\textcolor{red}{For this tutorial, however, we will be using the
+\lstinline{rose suite-run} command to run suites}.  It automatically installs,
+registers, validates, starts the suite, and opens the cylc GUI.  Rose
+is installed on your metomi VM. 
+
+\more{ \lstinline{rose suite-run} will soon be migrated into cylc.  It is a
+    convenient short-cut command, but more importantly it {\em installs} the
+    suite into its run directory and registers it there.  This separates the
+    live suite from its source, and provides an opportunity to install external
+files into the suite at start-up.  }
 
-Next copy the ``hello world example'' code into a suite.rc file within that
-directory.
-
-Finally run the following commands:
+\paragraph*{Demo}
+Copy the ``hello world example'' into a suite.rc file in a directory called
+hello\_world, and run the following commands:
 
 \begin{lstlisting}[language=bash]
-$ cd hello_world  # move to the suite directory
+$ cd hello_world
 $ touch rose-suite.conf  # create an empty rose-suite.conf file
 $ rose suite-run  # run the suite
 \end{lstlisting}
 
-If successful cylc will generate some output then a window will open. This is
-the cylc monitoring and control tool which displays the current status of
-your suite. In the cylc gui you should briefly see the task
-\lstinline{hello_world}
-displayed with a coloured square next to it. The colour of the square
-represents the state of the task
-(for example green means the task is running and gray that it has succeeded).
-Once the hello\_world task has succeeded, having no more tasks to run the suite
-will automatically shut down.
+If successful cylc will generate some output and open the cylc monitoring and
+control GUI, which displays the current status of your suite. You should
+briefly see the task \lstinline{hello_world} displayed with a coloured square
+that represents its state. For example green means the task is running and gray
+that it has succeeded.  Once the task has succeeded, the suite has no more
+tasks to run and will automatically shut down.
+
+The task job script and its standard out and standard error are captured to
```

Comment:
This seems a bit heavy for hello_world, could It be moved to a later point or into the advanced tutorials section?